src/taskgraph/run-task/run-task

```diff
@@ -1033,6 +1033,33 @@ def _display_python_version():
         b"setup", b"Python version: %s\n" % platform.python_version().encode("utf-8")
     )
 
+def _display_subprocess_python_version():
+    try:
+        output = subprocess.check_output(
+            ["uv", "python", "list", "--only-installed"],
+        ).strip()    
+
+        if output:
+            for line in output.splitlines():
+                print_line(b"setup", b"Subprocess python version: " + line + b"\n")
```

Comment:
I don't see any value in prefixing the uv output here. Was thinking you could do a pre-amble log, maybe something like:
```
print_line(b"setup", b"Detected the following Python versions on the system:")
subprocess.run(["uv", ...])
```

(i.e, don't capture the output and just let it pass through to stdout)